libs/@local/hash-backend-utils/src/pg-tables.ts

```diff
@@ -2,74 +2,106 @@
  * This module contains type definitions for tables in the Postgres database. Column
  * names are converted from snake_case to camelCase for consistency.
  */
+import { SupportedRealtimeTable } from "./realtime";
 import { Wal2JsonMsg } from "./wal2json";
 
-export class Entity {
-  constructor(
-    /**
-     * @todo Update table definition when provenance info is added for updates
-     *   https://app.asana.com/0/1202805690238892/1202848989198291/f
-     */
-    public entityId: string,
-    public version: string,
-    public entityTypeVersionId: string,
-    public properties: Record<string, unknown>,
-    public createdBy: string,
-  ) {}
+export type EntityEdition = {
```

Comment:
Kinda want to insert :panic: emoji here.. How many times are we going to redefine these things and have separate sources of truth 😭

Reply:
Yeah, it's definitely not ideal. And as you've commented below, it's very easy to use the wrong types. Not sure how we can make this better without a codegen step from the SQL/DB to rust/ts/etc.